metadata-ingestion/source_docs/metabase.md

```diff
@@ -0,0 +1,58 @@
+# Superset
+
+For context on getting started with ingestion, check out our [metadata ingestion guide](../README.md).
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[metabase]'`.
+
+See documentation for Metabase's API at https://www.metabase.com/learn/administration/metabase-api.html 
+for more details on Metabase's login api.
+
+
+## Capabilities
+
+This plugin extracts the following:
+
+- Charts, dashboards, and associated metadata
```

Comment:
What about lineage from Charts to upstream Datasets?

Reply:
Let's note it explicitly.

Reply:
fixed. Please see [Documentation](https://github.com/linkedin/datahub/blob/0c03ae4bc9ca7e473bb7c5c7f0ab6f3a1fb446ab/metadata-ingestion/source_docs/metabase.md)

Reply:
fixed. Please see [Documentation](https://github.com/linkedin/datahub/blob/0c03ae4bc9ca7e473bb7c5c7f0ab6f3a1fb446ab/metadata-ingestion/source_docs/metabase.md)